page-object.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'selenium-webdriver', '>= 2.44.0'
   s.add_dependency 'page_navigation', '>= 0.9'
 
-  s.add_development_dependency 'rspec'
+  s.add_development_dependency 'rspec', '~> 3.1.0'
```

Comment:
this was to prevent test breakage. Now that there is a fix for that in master, this can be omitted.